driftbase/schemas/players.py

```diff
@@ -83,3 +88,16 @@ def populate_urls(self, obj, many=False):
         )
         return obj
 
+    @pre_dump
+    def populate_total_match_time(self, obj, many=False):
+        match_time_query = g.db.query(func.sum(MatchPlayer.leave_date - MatchPlayer.join_date)).filter(MatchPlayer.player_id == obj.player_id)
```

Comment:
On top of that there's already a field "seconds" which contains the result of the above calculation for any finished match.

Reply:
I updated the query to sum up the seconds column.

Caching this would be an optimization and adds a layer of complexity.
This approach is probably the simplest time to market solution.